gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -813,6 +836,55 @@ export function GeneralSettings() {
               settingType="general"
               id="mechanics-magnetometer"
             />
+            <div className="flex flex-col pt-4 pb-4"></div>
+            <Typography bold>
+              {l10n.getString(
+                'settings-general-tracker_mechanics-spine_yaw_correction'
+              )}
+            </Typography>
+            <div className="flex flex-col pt-2 pb-4">
+              {l10n
+                .getString(
+                  'settings-general-tracker_mechanics-spine_yaw_correction-description'
+                )
+                .split('\n')
+                .map((line, i) => (
+                  <Typography color="secondary" key={i}>
+                    {line}
+                  </Typography>
+                ))}
+            </div>
+            <div className="grid sm:grid-cols-1 gap-3 pb-4">
+              <CheckBox
+                variant="toggle"
+                outlined
+                control={control}
+                name="yawCorrectionSettings.enabled"
+                label={l10n.getString(
+                  'settings-general-tracker_mechanics-spine_yaw_correction-enabled-label'
+                )}
+              />
+              <div>
+                <Typography bold>
+                  {l10n.getString(
+                    'settings-general-tracker_mechanics-spine_yaw_correction-amount-label'
+                  )}
+                </Typography>
+                <Typography color="secondary">
+                  {l10n.getString(
+                    'settings-general-tracker_mechanics-spine_yaw_correction-amount-description'
+                  )}
+                </Typography>
+                <NumberSelector
+                  control={control}
+                  name="yawCorrectionSettings.amountInDegPerSec"
+                  valueLabelFormat={(value) => degreeFormat.format(value)}
```

Comment:
This label should say in degrees per second, not just degrees. It is confusing as it is.

Reply:
See https://github.com/SlimeVR/SlimeVR-Server/pull/1243#discussion_r1866370498